### ISSUE TITLE
Bag screen doesn't scroll if you have too many items

### ISSUE THREAD

**Issue by ultidonki** [Tuxemon/Tuxemon#2405]
See screenshot: I'm not sure how hard this is to fix. And it's not really a big deal as I think you can still examine and use all items.
![item_menu](https://github.com/user-attachments/assets/75b4f1d9-136e-488e-9d2d-bb82380530d1)

Save file to reproduce the issue (Well, it's the exact same save file from the Tuxepedia crash issue :smile: ):
[slot3_bag_full.zip](https://github.com/user-attachments/files/16899760/slot3_bag_full.zip)

**Comment by JaskRendix** [comment]
this one is an old one, way before 2022 when I started wasting time on Tuxemon, I think there is another issue somewhere, we need to refactor the inventory

**Comment by ultidonki** [comment]
Thanks for the info - I can't see another issue for this, so I'll leave this one open.

This could possibly be fixed by changing the tuxemon/states/items/item_menu.py to use the pygame_menu module. But there's quite a few sub-menus in that file, and I don't think we have an infinitely-scrollable menu elsewhere in the game yet, so maybe too complex for now. It's low priority for me anyway, I'll leave it here as something I noticed.

**Comment by JaskRendix** [comment]
@ultidonki  exactly, that's the idea - having everything integrated into pygame_menu, like the player menu and tuxepedia. I tried to tackle the actual code yesterday, but I only managed to set the area, and it's still not perfect. The issue is that it cuts off two items, 'Tea' and 'Super Potion', because it's missing a scroll feature. I'm still working on figuring that part out. The same issue would happen with the Techniques too, but since these are limited to 4 per monster, it'll never happen.

> I don't think we have an infinitely-scrollable

technically the monster boxes and item boxes, both with pygame_menu
